src/main/scala/tv/teads/wiremock/extension/FreeMarkerRenderer.scala

```diff
@@ -79,4 +83,44 @@ class FreeMarkerRenderer extends ResponseDefinitionTransformer {
       hash
   }
 
+  class FindFirstInArray(requestBody: JsonNode) extends TemplateMethodModelEx {
+    import freemarker.template.TemplateModelException
+
+    @tailrec
+    private def findChildNode(parent: JsonNode, fullPath: String): JsonNode = {
+      val childNodes = fullPath.split('.')
+      if (childNodes.length == 1)
+        parent.findPath(fullPath)
+      else findChildNode(parent.findPath(childNodes.head), childNodes.tail.mkString("."))
+    }
+
+    private def extractPathAndValueFromCondition(filteredChildCondition: String): (String, String) = {
+      val splitFilteredChildCondition = filteredChildCondition.split("==").map(_.trim)
```

Comment:
mmm. I guess that it could be more generic by using JsonPath here, with the correct lib.

Reply:
This is exactly the Json path filter i wanted to do : https://github.com/json-path/JsonPath/issues/272 . But as you can see, it is not possible as of now with JsonPath syntax. That is why i use a custom method to handle this problem.

Reply:
OK as a workaround